spec/controllers/phone_numbers_controller_spec.rb

```diff
@@ -81,8 +83,15 @@
 
   describe "PUT #update" do
     context "with valid params" do
+      let(:bob) { Person.create(first_name: "Bob", last_name: "Smith") }
+
+
+      let(:valid_attributes) {
+        { number: '555-5678', person_id: bob.id }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.